h/accounts/test/models_test.py

```diff
@@ -6,20 +6,21 @@
 import pytest
 from sqlalchemy import exc
 
+from h import db
 from h.accounts import models
 from h.test import factories
 
 
-def test_activation_has_asciinumeric_code(db_session):
```

Comment:
The database session is not injected or configured explicitly in application code, so I'm not sure why we'd necessarily want it to be so in the tests. I agree that there's a "too much magic" line for test setup/teardown code, but I don't think this is on the far side of it -- there's a bunch of other code that is `autouse`d and I don't think we'd like what our tests looked like if we had to explicitly invoke those fixtures for every test.